From 7ff94b874cb22438b654120a5990a7469a5b5d2b Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 24 Jul 2014 14:21:15 +0000
Subject: [PATCH 0482/1000] MANTIS 0033134 extension Trace DLL: add description
 of trace format

trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll
V1.0.0.6

- relative path: "<year>/<month>/<day>"
- file name:     "applicationname_<year><month><day>_<hour><minute><second>_<milliseconds>_<pid>.trc"
- line:          "<day>.<month>.<year>\t<hour>:<minute>:<second>,<milliseconds>\t<thread index>\t<level/topic>\t<file>(<line>)"\n
  for "Trace Init" and "Trace Close" the thread id is used; because the thread index cannot be calculated
- dump:          "D\t\t<dec position>\t<hex position>\t<hex data>\t<data>"
the applications use proprietary line formats using writeTraceCooked



git-svn-id: svn://localhost/SelfServiceCommon/trunk@701 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Trace/inc/local_def.h            | 17 ++++++++++++++---
 1 file changed, 14 insertions(+), 3 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
index a702b050..1969decd 100644
--- a/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
+++ b/SelfServiceCommon/Massai/cpp/Trace/inc/local_def.h
@@ -1,5 +1,5 @@
-#define FILE_VER_BIN     1,0,0,5
-#define FILE_VER_STR     "FileVersion", "1.0.0.5\0"
+#define FILE_VER_BIN     1,0,0,6
+#define FILE_VER_STR     "FileVersion", "1.0.0.6"
 
 /*!
 \file
@@ -9,10 +9,21 @@
 
 \section changehistory Change history
 
-@version 1.0.0.5
+\version 1.0.0.6
 
 trace.dll, tracehdl.dll, traceNoConfig.dll, tracehdlNoConfig.dll
 
+\version 1.0.0.6
+\li 24.07.2014
+\li extension SteT - Mantis 0033134: Trace DLL: add description of trace format
+- relative path: "<year>/<month>/<day>"
+- file name:     "applicationname_<year><month><day>_<hour><minute><second>_<milliseconds>_<pid>.trc"
+- line:          "<day>.<month>.<year>\t<hour>:<minute>:<second>,<milliseconds>\t<thread index>\t<level/topic>\t<file>(<line>)"\n
+  for "Trace Init" and "Trace Close" the thread id is used; because the thread index cannot be calculated
+- dump:          "D\t\t<dec position>\t<hex position>\t<hex data>\t<data>"
+the applications use proprietary line formats using writeTraceCooked
+
+
 \version 1.0.0.5
 \li 23.10.2013
 \li bugfix -  Mantis 0029551: Buildsystem: Integrate Boost 1.54
-- 
2.41.0.windows.1

